src/android/Screenshot.java

```diff
@@ -70,11 +70,7 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
No need for the conditional statement here I think, just call `saveScreenshot` @attilakiss323